activity.py

```diff
@@ -16,51 +31,52 @@
 from sugar3.activity.widgets import StopButton
 from game_engine import Game
 
+BUTTON_LABELS = ("Easy", "Medium", "Hard")
+MENU_BACKGROUND_PATH = 'backgrounds/menu_background.png'
+GAME_BACKGROUND_PATH = 'backgrounds/background{}.png'
+MAP_PATH = 'map{}.json'
+BOX_SPACING = 6
+
 
 class TextdungeonActivity(activity.Activity):
     def __init__(self, handle):
-        activity.Activity.__init__(self, handle)
+        super().__init__(handle)
```

Comment:
A sugar activity must implement a new class derived from `sugar3.activity.activity.Activity`, why'd you change this?

Reply:
The inheritance hasn't changed, only the method of calling the parent's __init__ has. Did I misunderstand something?